src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    gap: 20px;
+    display: flex;
+    margin: 0;
+    padding: 0;
+    flex-direction: row;
+    list-style: none;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 60px;
+    text-decoration: none;
+    font-size: 12px;
+    color: $color-link;
+    font-weight: 500;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $color;
+      transition: color 300ms ease-in-out;
+    }
+  }
+
+  &__item {
+    position: relative;
+  }
+}
+
+.is-active {
+  color: $color;
+
+  &::after {
+    content: '';
+    bottom: 0;
+    left: 0;
+    position: absolute;
+    background-color: $color;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+  }
+}
```

Comment:
The `.is-active` class should be combined with the element it modifies, for example, `.nav__link.is-active` to increase specificity and to ensure that it's clear what element this modifier class is associated with.